admin/multiadd.php

```diff
@@ -17,55 +17,76 @@
 	}
 
 	if (mysqli_num_rows ( $series_list ) > 0) {
-		$dropdown = "<select class=\"element select medium\" id=\"element_6\" name=\"series_name\">\n";
-		$dropdown .= "<option value=\"\" selected=\"selected\"></option>\n";
+		$dropdown = '<select class="form-control" id="element_6" name="series_name">';
```

Comment:
It's semantic and totally a personal preference, but I like to use single quotes for PHP strings. It allows you to use "" inside without have to escape characters which is just ugly to look at.